back/babble/src/main/java/gg/babble/babble/dto/RoomRequest.java

```diff
@@ -17,4 +18,6 @@
     private Long gameId;
     @NotNull(message = "태그 목록은 Null 일 수 없습니다.")
     private List<TagRequest> tags;
+    @Min(value = 2, message = "방 최대 참가 인원 최소 2인 이상이어야 합니다.")
```

Comment:
```suggestion
    @Min(value = 2, message = "최소 2인 이상 방에 참가 가능합니다.")
    @Max(value = 20, message= "최대 20인 이하 방에 참가 가능합니다.")
```

문맥이 이상하네요, 양방향 검증을 할 필요가 있어 보이네요.